components/text/tidy/Makefile

```diff
@@ -21,62 +21,40 @@
 
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2025 Klaus Ziegler
 #
 
+BUILD_STYLE= cmake
+USE_PARALLEL_BUILD= yes
 include ../../../make-rules/shared-macros.mk
```

Comment:
Please add
DROP_STATIC_LIBRARIES = yes
here. This should change the sample-manifest.